### ISSUE TITLE
add setAlwaysOnTop() to surface

### ISSUE THREAD

**Issue by Leappeal** [processing/processing#3718]
"frame.setAlwaysOnTop(true);" no longer works in the 3.0 version. Has anyone figured out a way to keep the window on top?

**Comment by benfry** [comment]
@codeanticode is this possible in JOGL?

**Comment by neilcsmith-net** [comment]
Definitely possible with JOGL.

My current workaround is -

```java
protected PSurface initSurface() {
            PSurface s = super.initSurface();
            GLWindow window = (GLWindow) surface.getNative();
            window.setAlwaysOnTop(true);
            // window.setUndecorated(true);
            return s;
}
```
Note also possible to do things like set undecorated that way.  This may be possible in other places than overriding initSurface(), but not sure all systems will take any notice once the window is visible.

**Comment by benfry** [comment]
Implemented for all renderers in 3.0 beta 6.
